language/analyze/README.md

```diff
@@ -0,0 +1,67 @@
+# Google Cloud Natural Language API Go example
+
+## Authentication
+
+* Follow the [instructions][project] to set up your project and enable the Cloud Natural Language API.
+* From the Cloud Console, create a service account,
+  download its json credentials file, then set the 
+  `GOOGLE_APPLICATION_CREDENTIALS` environment variable:
+
+  ```bash
+  export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
+  ```
+
+[cloud-console]: https://console.cloud.google.com
+[project]: https://cloud.google.com/natural-language/docs/getting-started#set_up_your_project
+
+## Run the sample
```

Comment:
Yes, I think so. I want to add that for all of the other samples, too, though, and to do it consistently. So I'll add that later.

All of the docs on cloud.google.com should suggest `go get`, though.